ca/ca.go

```diff
@@ -447,9 +456,12 @@ func (ca *CertificateAuthorityImpl) IssueCertificateForPrecertificate(ctx contex
 	return emptyCert, berrors.InternalServerError("IssueCertificateForPrecertificate is not implemented")
 }
 
-func (ca *CertificateAuthorityImpl) generateNotAfterAndSerialNumber() (time.Time, *big.Int, error) {
-	notAfter := ca.clk.Now().Add(ca.validityPeriod)
+type Validity struct {
```

Comment:
Is there a reason to export this type outside of the `ca` package or could it be renamed `validity`?